orm/tooling.adoc

```diff
@@ -68,7 +68,7 @@ To use Hibernate Metamodel Generator add the following dependency to your build
 </dependency>
 ----
 
-For more information and setup options refer to the link:https://docs.jboss.org/hibernate/orm/{version_family}/topical/html_single/metamodelgen/MetamodelGenerator.html[online documentation].
+For more information and setup options refer to the link:https://docs.jboss.org/hibernate/orm/{version_family}/userguide/html_single/Hibernate_User_Guide.html#tooling[online documentation].
```

Comment:
probably the link should be https://docs.jboss.org/hibernate/orm/6.1/userguide/html_single/Hibernate_User_Guide.html#tooling-modelgen